extensions/objects/protractor.js

```diff
@@ -30,11 +30,24 @@ var AlgebraicExpressionEditor = function(elem) {
       await waitFor.elementToBeClickable(
         elem, '"Algebraic Input" editor takes too long to be clickable');
       await elem.click();
-      var algebraicInputElem = element(by.css('.guppy_active'));
+      var algebraicInputElem = element.all(by.css(
+        '.protractor-test-guppy-div')).last();
```

Comment:
Normally we discourage using `.last()`, because it tends to make tests dependent on each other, but in this case we actually know that the element we want will be last (https://github.com/oppia/oppia/pull/9553#discussion_r445905237), so it's fine. Could you just add a comment explaining that we know the active element will be last?

Reply:
Done.